service/src/main/java/com/codeforcommunity/requester/S3Requester.java

```diff
@@ -369,4 +377,28 @@ public static void deleteHTML(String name) {
 
     externs.getS3Client().deleteObject(awsRequest);
   }
+
+  public static List<String> getAllNamesinBucket(String bucketName) {
+    ListObjectsV2Request req = new ListObjectsV2Request();
+    req.setBucketName(bucketName);
+    req.setDelimiter("/");
+    String path = String.join("/", TEMPLATE_S3_DIR, externs.getDirPublic());
```

Comment:
🔧 When I tried this route, it only returned an empty list until I added "/" to the end of the path - were you able to get template names without it?

Reply:
fixed and tested